### PR TITLE
`azurerm_managed_disk` - Support `optimized_frequent_attach_enabled` and `performance_plus_enabled`

### PR DESCRIPTION
**Pull request by ms-zhenhua**
## Related Issue
Fix https://github.com/hashicorp/terraform-provider-azurerm/issues/21388

## Test Result

image

### CODE REVIEW

File: website/docs/r/managed_disk.html.markdown [hunk 1]
```diff
@@ -133,6 +133,10 @@ The following arguments are supported:
 
 ~> **NOTE:** Setting logical sector size is supported only with `UltraSSD_LRS` disks and `PremiumV2_LRS` disks.
 
+* `optimized_frequent_attach_enabled` - (Optional) Whether the feature of improving reliability and performance of data disks that are frequently (more than 5 times a day) detached from one virtual machine and attached to another is enabled. This property should not be set for disks that are not detached and attached frequently as it causes the disks to not align with the fault domain of the virtual machine. Defaults to `false`.
```
Comment (tombuildsstuff) [inline website/docs/r/managed_disk.html.markdown:136]:
```suggestion
* `optimized_frequent_attach_enabled` - (Optional) Specifies whether this Managed Disk should be optimized for frequent disk attachments (where a disk is attached/detached more than 5 times in a day). Defaults to `false`.

-> **Note:** Setting `optimized_frequent_attach_enabled` to `true` causes the disks to not align with the fault domain of the Virtual Machine, which can have operational implications.
```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
hey @ms-zhenhua 

Thanks for this PR - I've taken a look through and left some comments inline, but if we can fix those up then this should otherwise be good to go 👍 

Thanks!

File: website/docs/r/managed_disk.html.markdown [hunk 2]
```diff
@@ -133,6 +133,10 @@ The following arguments are supported:
...
+
+* `performance_plus_enabled` - (Optional) Whether performance plus is enabled for this disk. Performance plus is only supported for UltraSSD disks. Defaults to `false`. Changing this forces a new resource to be created.
```
[the remark this answers is not in the document]
Comment (ms-zhenhua) [reply website/docs/r/managed_disk.html.markdown]: Doc updated. Thanks.

**Comment by ms-zhenhua** [comment]
Hi @tombuildsstuff, thank you for your review. I have resolved all the comments. Could you kindly help take another review? Thanks.

**Review by tombuildsstuff — APPROVED** [review]
LGTM 👍

**Pull request merged**